piglow/examples/strobe.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"time"
+
+	"github.com/goiot/devices/piglow"
+	"golang.org/x/exp/io/i2c"
+)
+
+func main() {
+	p, err := piglow.Open(&i2c.Devfs{Dev: "/dev/i2c-1", Addr: 0x54})
```

Comment:
Yes, libraries will be able to embed addresses once https://go-review.googlesource.com/#/c/24215/ is merged.